### PR TITLE
Add fonts, colors, typeScale

### PR DESCRIPTION
**Pull request by PhuongPhg**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,6 +16,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="preloaded_fonts"
+            android:resource="@array/preloaded_fonts" />
+
```
Comment (McSinyx) [inline app/src/main/AndroidManifest.xml:22]: Genuine question, what does this do?

**Review by McSinyx — COMMENTED** [review]
Does the font work with Vietnamese (and other common non-ascii latin) characters, e.g. *Trịnh Thảo Vân*?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (9a24f0) [reply app/src/main/AndroidManifest.xml:22]: I think [this guide](https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts#predeclaring-fonts) helps.

Comment (McSinyx) [reply app/src/main/AndroidManifest.xml:22]: Thanks!

**Comment by PhuongPhg** [comment]
> Does the font work with Vietnamese (and other common non-ascii latin) characters, e.g. _Trịnh Thảo Vân_?

No 😢 we can use Montserrat Alternates instead but i prefer poppins :(

**Comment by Huy-Ngo** [comment]
Nits: It doesn't support non-latin characters, such as Greek, Chinese, Arabic, devanagari, etc. GitHub has clients from various countries and there name may contain those characters so we better have those.

For example, a popular JS lib such as [Vue.JS](https://github.com/search?q=vue&ref=opensearch) can contains a lot of Chinese comments, issues, description. I'd rather have them displayed correctly than square square thingy.

**Comment by McSinyx** [comment]
-1 from the having-enough-sleep me (and from dr dx as well).  There are very few fonts that can cover the whole Unicode range (e.g. Free Sans, but we can all agree that it looks like crap) and maintaining typeface fallback with visual consistency is no more trivial than making the UI independent of font choice.  I'd prefer to respect the users' preferences—one would choose a font that support the character-range of what perse can read already.

**Comment by Huy-Ngo** [comment]
We can still merge those files for color and size.

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,6 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!--?xml version="1.0" encoding="UTF-8"?-->
```
Comment (McSinyx) [inline app/src/main/res/values/colors.xml:1]: Why was this commented out?  Otherwise LGTM.

Comment (PhuongPhg) [reply app/src/main/res/values/colors.xml:1]: I think the that line would not affect  so i commented out in case it's needed

Comment (McSinyx) [reply app/src/main/res/values/colors.xml:1]: Isn't it what makes an XML a valid one?

Comment (PhuongPhg) [reply app/src/main/res/values/colors.xml:1]: i think it's not necessary because other .xml file in values doesn't include that too

**Review by McSinyx — APPROVED** [review]
Fair enough.

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (Huy-Ngo) [reply app/src/main/res/values/colors.xml:1]: Wot why comment it then?

**Pull request merged**